packages/opc-header/src/opc-header.scss

```diff
@@ -0,0 +1,97 @@
+@import "@one-platform/opc-styles";
+
+$LOCAL: opc-header;
+
+$variables: (
+  //Add your variables here
+);
```

Comment:
@diwanshi should we remove this because we are using :host
```suggestion
```

Reply:
umm we are gonna need this soon as we work on variable naming automation. do u think we should still remove it?

Reply:
Ohh okay, cause there is a function in opc-styles known as opc-get-var(). @gisumit Do we have examples?